src/models/cost_function_timeseries.jl

```diff
@@ -0,0 +1,112 @@
+function get_variable_cost(
+    ts::IS.TimeSeriesData,
+    ::Union{MarketBidCost, ReserveDemandCurve},
+)
+    data = TimeSeries.TimeArray(TimeSeries.timestamp(ts), map(VariableCost, ts.data))
+    return data
+end
+
+function get_variable_cost(
```

Comment:
We need to add something here that catches the case the key in the `MarketBidCost` is nothing